### ISSUE TITLE
Offset-Dropdown

### ISSUE THREAD

**Issue by mega-stoffel** [Friends-of-OpenBikeSensor/OpenBikeSensorFirmware#86]
in der Konfiguration sollte beim Auswählen des Offsets ein DropDown-Menü vorhanden sein, bei dem man einfach nur seine eigenen Räder auswählen kann und die Offset-Werte dann automatisch in die Konfiguration übernommen werden.
Somit muss man nicht mehr ständig nachmessen und kommt bei mehreren Rädern nicht durcheinander.

**Comment by boldt** [comment]
@mega-stoffel - Wäre #101 eine alternative Lösung?

**Comment by mega-stoffel** [comment]
ja, ich denke mit diesem Down/Upload der Konfiguration ist es für den Anfang mal ok.
